AUTHORS.md

```diff
@@ -0,0 +1,55 @@
+HardInfo was started in 2003, by Leandro Pereira, and has received
+contributions from many people over the years.
+
+There was a time it didn't even use a version control system. The early
+notes and changelogs were lost. Some important contributions were made
+by simply submitting bug reports or comments. For these reasons, it
+would be difficult to gather the names of all the people that
+contributed to the project.
+
+Here is an incomplete list of contributors:
+
+* Leandro A. F. Pereira <leandro@hardinfo.org>
+* Stewart Adam <s.adam@diffingo.com>
+* Pascal F. Martin <pascalmartin@earthlink.net>
+* Burt P. <pburt0@gmail.com>
+* TotalCaesar659 <x24cm5b8c54q6szxw@yandex.ru>
+* Andrey Esin <gmlastik@gmail.com>
+* Julian Ospald <julian.ospald@googlemail.com>
+* Julien Lavergne <gilir@ubuntu.com>
+* Fernando López <fernando.e.lopez@gmail.com>
+* PICCORO Lenz McKAY <mckaygerhard@gmail.com>
+* Alexander Münch <git@thehacker.biz>
+* Simon Quigley <tsimonq2@ubuntu.com>
+* AsciiWolf <mail@asciiwolf.com>
+* George Schneeloch <george@editshare.com>
+* Mattia Rizzolo <mattia@mapreri.org>
+* Yo <yoldc22@free.fr>
+* "jamesbond"
```

Comment:
I unfortunately don't have the contact information for this `jamesbond` person.  I've taken their patches from downstream here: https://chiselapp.com/user/jamesbond/repository/hardinfo/home